lib/asset_sync/engine.rb

```diff
@@ -44,7 +44,8 @@ class Engine < Rails::Engine
           config.enabled = (ENV['ASSET_SYNC_ENABLED'] == 'true') if ENV.has_key?('ASSET_SYNC_ENABLED')
 
           config.existing_remote_files = ENV['ASSET_SYNC_EXISTING_REMOTE_FILES'] || "keep"
-          config.gzip_compression = (ENV['ASSET_SYNC_GZIP_COMPRESSION'] == 'true') if ENV.has_key?('ASSET_SYNC_GZIP_COMPRESSION')
+          config.compression = 'gz' if ENV.has_key?('ASSET_SYNC_GZIP_COMPRESSION') and ENV['ASSET_SYNC_GZIP_COMPRESSION'] == 'true'
+          config.compression = ENV['ASSET_SYNC_COMPRESSION'] if ENV.has_key?('ASSET_SYNC_COMPRESSION')
```

Comment:
```suggestion
          config.compression = 'gz' if ENV.has_key?('ASSET_SYNC_GZIP_COMPRESSION') and ENV['ASSET_SYNC_GZIP_COMPRESSION'] == 'true'
          config.compression = ENV['ASSET_SYNC_COMPRESSION'] if ENV.has_key?('ASSET_SYNC_COMPRESSION')
```

Reply:
There doesn't seem to be any changes? 
<img width="276" alt="image" src="https://github.com/AssetSync/asset_sync/assets/219422/d6518837-7819-4c17-a84d-684116aef675">

Reply:
Updated
I meant to change `and` > `&&`